### PR TITLE
correct javaDoc (#105)

### PR DESCRIPTION
**Pull request by ilyeskallala**
* correct javaDoc

* correct javaDoc

* correct java doc

### CODE REVIEW

File: wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/WikiValidattions.java [hunk 1]
```diff
@@ -137,7 +137,8 @@ public void verifyWarningMessage(String mess) {
   /**
    * Verify effects of Page's content
    *
-   * @param type String
+   * @param type  String
+
```
Comment (thomasdelhomenie) [inline wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/WikiValidattions.java]: Use less line.

File: forum/src/main/java/org/exoplatform/platform/qa/ui/forum/pageobject/ForumTopicManagement.java [hunk 2]
```diff
@@ -489,7 +489,7 @@ public void replyTopic(String newTitle, String newMessg, String pathFile, String
    *
    * @param title String
    * @param message String
-   * @param fileName String
+   *@param fileName String
```
Comment (thomasdelhomenie) [inline forum/src/main/java/org/exoplatform/platform/qa/ui/forum/pageobject/ForumTopicManagement.java]: Useless change.

**Comment by ilyeskallala** [comment]
@thomasdelhomenie, if we remove platform/src/test/resources/data, the submodule will be deleted. it contains a link to platform-qa-ui-data

**Comment by ilyeskallala** [comment]
@thomasdelhomenie ah, I added it manually because I did a "cherry-pick" of an old commit of correct java-doc and this submodule was deleted after this cherry-pick.

**Pull request merged**